finagle/h2/src/main/java/io/netty/handler/codec/http2/BetterHttp2ConnectionEncoder.java

```diff
@@ -28,6 +28,7 @@
 import static io.netty.handler.codec.http2.Http2CodecUtil.DEFAULT_PRIORITY_WEIGHT;
 import static io.netty.handler.codec.http2.Http2Error.PROTOCOL_ERROR;
 import static io.netty.handler.codec.http2.Http2Exception.connectionError;
+import static io.netty.handler.codec.http2.Http2FrameWriter.Configuration;
```

Comment:
Unless you feel strongly I'd prefer to leave this as is, just because it's annoying to test.